charts/velero/values.yaml

```diff
@@ -61,9 +61,18 @@ initContainers: []
 # SecurityContext to use for the Velero deployment. Optional.
 # Set fsGroup for `AWS IAM Roles for Service Accounts`
 # see more informations at: https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html
-securityContext: {}
+podSecurityContext: {}
   # fsGroup: 1337
 
+# Container Level Security Context for the 'velero' container of the Velero deployment. Optional.
+# See: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+containerSecurityContext: {}
+  # allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop: ["ALL"]
+  #   add: []
```

Comment:
I'm not sure should we list these values in the values.yaml?
I believe it does not include the full security context.
If not, probably we could consider removing it. WDYT?

Reply:
These are settings most users want to set for security reasons. But I did not want to change defaults to not break any edge-case setup.